### ISSUE TITLE
Is Greenshot screenshot tool for windows discontinued (as I cannot see any updated since 2017)?

### ISSUE THREAD

**Issue by ghost** [greenshot/greenshot#533]
Is Greenshot screenshot tool for windows discontinued (as I cannot see any updated since 2017)?

This GitHub repository is for developers, if you want to report a bug for Greenshot as a user please do so in our JIRA issue system here: https://greenshot.atlassian.net/projects/BUGS/issues/filter=allopenissues

**Describe the bug**

**To Reproduce**
Example steps to reproduce the behavior:
1. Capture '....'
2. Open Editor
3. Add a line
4. Resize

**Expected behavior**

**Screenshots**

**Versions (please complete the following information):**
 - Greenshot version
 - Windows version

**Additional context**

**Comment by ciarpame** [comment]
Unstable versions works fine for me, the last one is dated March 2023, you can download it from here: https://github.com/greenshot/greenshot/releases

**Comment by Varniit-pro-90s** [comment]
> Unstable versions works quite fine for me (some minor bearable bugs), the last one is dated March 2023, you can find it here: https://github.com/greenshot/greenshot/releases To download it, expand "Assets" at the bottom of the entry and you will file the installer link. However, as you can read from the other posts, there is some concern about Greenshot's future.

But original website states only 2017 not any date after that. Shouldn't original website have this info too if it recommended and used by many?

What about to GS dev and team?

**Comment by jklingen** [comment]
Hi folks,
sorry, everything's going slowly for quite some time now 😕  We just posted a new blog post about the [current status of Greenshot](https://getgreenshot.org/2024/02/11/current-status-greenshot/).
I hope it adds some clarity.

> tl;dr: We are struggling with a lack of time, but are gradually trying to pick up speed again.

**Comment by jklingen** [comment]
> Shouldn't original website have this info too

We have this blue box on our [downloads page](https://getgreenshot.org/downloads/), maybe not prominent enough 🤔  we just do not want to _generally_ recommend unstables releases _to everyone_.

**Comment by Varniit-pro-90s** [comment]
> > Shouldn't original website have this info too
> 
> We have this blue box on our [downloads page](https://getgreenshot.org/downloads/), maybe not prominent enough 🤔 we just do not want to _generally_ recommend unstables releases _to everyone_.

On the same link you shared and I also have, there is no indication of updates only the highlighted date which doesnt promise any update for an average person as other websites show.

![2024-02-12 22 57 03](https://github.com/greenshot/greenshot/assets/156216269/946075ff-a803-4981-968f-5a199ff0ce8d)

**Comment by Varniit-pro-90s** [comment]
> @Varniit-pro-90s the latest _stable_ release is really from 2017, that is correct. Quite a few _unstable_ releases have been published since then. We do not generally recommend using unstable releases for average users, but the blue box below the prominent buttons points to our [version history](https://getgreenshot.org/version-history/) page which contains all releases.

In that case, an update is still needed for exe or msi and window app store version in 2024.

**Comment by rubcap** [comment]
@jklingen Take your time, guys. Greenshot is a wonderful tool.

**Comment by jklingen** [comment]
In case you haven't noticed: we have published Greenshot 1.3 last month: https://getgreenshot.org/2025/07/14/final-release-greenshot-1-3/
